ethereum/core/src/test/java/org/hyperledger/besu/ethereum/transaction/TransactionSimulatorTest.java

```diff
@@ -565,8 +565,9 @@ public void shouldCapGasLimitWhenOriginalTransactionExceedsGasCap() {
   }
 
   @Test
-  public void shouldKeepOriginalGasLimitWhenCapIsHigherThanOriginalValue() {
-    // generate a transaction with a gas limit that is lower than the gas cap
+  public void shouldUseRpcGasCapWhenCapIsHigherThanGasLimit() {
```

Comment:
what if cap is lower than gasLimit

Reply:
yeah, this test can really just be renamed to something like when present rpc gas cap supersedes request gas limit